src/main/java/nextstep/courses/domain/Session.java

```diff
@@ -0,0 +1,41 @@
+package nextstep.courses.domain;
+
+import nextstep.payments.domain.Payment;
+
+import java.time.LocalDate;
+import java.util.HashSet;
+import java.util.Set;
+public abstract class Session {
+    protected Long id;
+    protected String name;
+    protected LocalDate startDate;
+    protected LocalDate endDate;
+    protected Image coverImage;
+    protected LectureStatus status;
+    protected Set<Long> registeredStudents = new HashSet<>();
+    public Session(Long id, String name, LocalDate startDate, LocalDate endDate, Image coverImage, LectureStatus status) {
```

Comment:
```suggestion
    protected Set<Long> registeredStudents = new HashSet<>();

    public Session(Long id, String name, LocalDate startDate, LocalDate endDate, Image coverImage, LectureStatus status) {
```

멤버변수와 생성자는 개행으로 구분짓는 것이 컨벤션입니다 😄